docs/_config.yml

```diff
@@ -67,4 +69,9 @@ plugins:
   - jekyll-redirect-from
   - jekyll-seo-tag
 
-exclude: ["deploy/includes"]
+exclude: 
+  - deploy/includes
+  - vendor
```

Comment:
I wanted this one - some ppl may install the vendor dependencies locally and not on the user home directory.
This will prevent any mistakes in deploy something we don't want.